src/bionics.cpp

```diff
@@ -1532,9 +1521,22 @@ bool player::install_bionics( const itype &type, int skill_level )
                                               difficult );
     }
 
-    if( !query_yn(
-            _( "WARNING: %i percent chance of genetic damage, blood loss, or damage to existing bionics! Install anyway?" ),
-            100 - chance_of_success ) ) {
+    std::map<body_part, int> issues = bionic_installation_issues( bioid );
```

Comment:
Can become:
```c++
const auto &issues = bionic_installation_issues( bioid );
```
I suppose the ampersand can be omitted, though (due to compiler's optimisation).